src/components/Layout/Header/SideNavContent.tsx

```diff
@@ -87,6 +87,10 @@ export const SideNavContent = ({ route, isCompact }: HeaderContentProps) => {
             <Text translation='common.privacy' />
           </Link>
         </HStack>
+        {/* "Hidden" link to the flags page */}
```

Comment:
remove this now that we have the alt-f (windows/linux) or option-f (macos) keyboard shortcut